### DISCUSSION TITLE
Cannot send report

### DISCUSSION THREAD

**Discussion by hibino054** [yasserfarouk/scml#108] [General]
There is one day before the submission deadline, but as you can see in the attached image, the submission is closed.
TeamID : 100, track: Std 
<img width="1187" alt="スクリーンショット 2022-06-16 22 46 20" src="https://user-images.githubusercontent.com/88707356/174084206-72f621f7-0bed-4c8e-a50b-80671b501fcb.png">

**Comment by yasserfarouk** [comment]
We faced some problems with the submission server. It is working properly now and you should be able to upload your report. Do not forget to press "update" after you are done
